### PR TITLE
Update notification channels in travis

### LINKED ISSUE #419: Update travis configurations to point to updated notification locations.
## Description
As a developer, I should be able to get the results (notifications) of Travis test in the right location. This is a problem because the former communication channels have been changed but not updated in the Travis configuration file.

## Acceptance Criteria
- [x] Add correct notification locations in `.travis.yml`

## Definition of Done
- [x] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
0.5 hours

### PR DESCRIPTION
**Pull request by Fenn-CS**
Signed-off-by: Fenn-25 <fenn25.fn@gmail.com>

### Description
Removed outdated communication channels in Travis configuration, added slack

Fixes #419 

### Type of Change:
- Quality Assurance

### How Has This Been Tested?
Non code change, tests pass

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

### CODE REVIEW

**Comment by MehaKaushik** [comment]
Can we send a email to the collaborators and the person who has the latest PR??? If this is not in the scope, can its pros and cons be discussed and accordingly decided whether or not to go ahead with the email thing.

File: .travis.yml [hunk 1]
```diff
@@ -22,19 +22,6 @@ script:
...
-    on_failure: change
-    use_notice: true
-  email: false
-  webhooks:
-    urls:
-      - https://webhooks.gitter.im/e/c403378d624b2391961e
-    on_success: change
-    on_failure: always
-    on_start: false
+  slack: '<account>:<token>#portal'
```
Comment (Monal5031) [inline .travis.yml]: Can we set it in travis environment variables and use it here?

Comment (Fenn-CS) [reply .travis.yml]: Yes we can good suggestion...

Comment (ExactlyMay) [reply .travis.yml]: @Fenn-CS Did you update this?

Comment (Fenn-CS) [reply .travis.yml]: @ExactlyMay  this is issue is pending for admin approval as you specified.

Comment (Fenn-CS) [reply .travis.yml]: @ExactlyMay everything is now sorted out, I am just waiting for slack integration details t complete this. :)

Comment (MehaKaushik) [reply .travis.yml]: @Fenn-CS did you incorporate the change suggested by Monal here ??

Comment (Fenn-CS) [reply .travis.yml]: @MehaKaushik that would be included with the token and account details.

**Comment by sammy1997** [comment]
@mayburgos what should we do with these GSoC PRs? @Fenn-CS can you update the PR with changes?

**Comment by sakshi1499** [comment]
@Fenn-CS Can you update the changes otherwise we'll close the PR?

**Comment by sakshi1499** [comment]
This PR is not required anymore.

**Pull request closed without merging**